src/utils/datoUtils.js

```diff
@@ -316,9 +316,10 @@ export const addWeeks = (date, numberOfWeeks) => {
 
 const ONE_WEEK_MILLIS = 7 * 24 * 60 * 60 * 1000;
 
-export const getWeeksSinceDate = (date) => {
+export const getWeeksSinceDate = (startDate, endDate) => {
```

Comment:
Jeg synes ikke det er så lett å skjønne hva denne gjør. "Weeks since" tolker jeg som fra dato til i dag. Men når det sendes inn to datoer tenker jeg at den skal gjøre det samme som "weeks between", ikke at den gjør noe fancy magi med "now" o.l. 🤔 Hva med å skille ut "finn eldste dato" i en egen metode og så finne uker mellom? Denne kan evt. få et navn som er mer spesifikt for et oppfølgingstilfelle, for da er det kanskje enklere å skjønne at tilfelle kan være i fortiden?

(Og brukes denne funksjonen flere steder? I så fall kan noe ha brukket 🙀 )

Reply:
Enig, jeg gjør noen endringer her. Godt poeng det med at jeg kunne brukket noe. Det tenkte jeg ikke på